### PR TITLE
Fix find command bug for social handle

### PR DESCRIPTION
**Pull request by bklimey**
image
image
image
image

Previously, there was a bug preventing finding by social handle.

May need to look into the constraints for Nationality and Tutorial Group in FindCommandParser. (The constraints are currently commented out as the test cases do not have these constraints in mind)

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/FindCommandParser.java [hunk 1]
```diff
@@ -118,10 +118,11 @@ public FindCommand parse(String args) throws ParseException {
         if (argMultimap.getValue(PREFIX_NATIONALITY).isPresent()
             && !argMultimap.getValue(PREFIX_NATIONALITY).get().trim().isEmpty()) {
             List<String> nationalityKeywords = argMultimap.getAllValues(PREFIX_NATIONALITY);
-
+            /*
             for (String nationality : nationalityKeywords) {
-                ParserUtil.parseName(nationality);
+                ParserUtil.parseNationality(nationality);
             }
+             */
```
Comment (kevinmingtarja) [inline src/main/java/seedu/address/logic/parser/FindCommandParser.java:125]: Is this commented code still needed? If not I think would be better to remove it

File: src/main/java/seedu/address/logic/parser/FindCommandParser.java [hunk 2]
```diff
@@ -135,10 +136,11 @@ public FindCommand parse(String args) throws ParseException {
         if (argMultimap.getValue(PREFIX_TUTORIAL_GROUP).isPresent()
             && !argMultimap.getValue(PREFIX_TUTORIAL_GROUP).get().trim().isEmpty()) {
             List<String> tutorialGroupKeywords = argMultimap.getAllValues(PREFIX_TUTORIAL_GROUP);
-
+            /*
             for (String tutorialGroup : tutorialGroupKeywords) {
-                ParserUtil.parseName(tutorialGroup);
+                ParserUtil.parseTutorialGroup(tutorialGroup);
             }
+             */
```
Comment (kevinmingtarja) [inline src/main/java/seedu/address/logic/parser/FindCommandParser.java:143]: same here

**Review by kevinmingtarja — APPROVED** [review]
Minor comment but otherwise LGTM!

File: src/main/java/seedu/address/logic/parser/FindCommandParser.java (hunk 1 above)
Comment (choonyongchan) [reply src/main/java/seedu/address/logic/parser/FindCommandParser.java:125]: Don't remove it. I'll see if i can resolve it later. Same for the one below

**Pull request merged**